go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/honeycombio/terraform-provider-honeycombio
 
 go 1.23.6
 
+toolchain go1.24.1
```

Comment:
Weird, why'd this get added 🤔

Reply:
Nice catch! 99% sure it shouldn't be there. I just removed it to ensure we stay on Go 1.23.x